ui/src/alerting/components/builder/ThresholdStatement.tsx

```diff
@@ -0,0 +1,169 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {
+  FlexBox,
+  Panel,
+  ComponentSize,
+  ComponentStatus,
+  AlignItems,
+  PanelBody,
+  TextBlock,
+  SelectDropdown,
+  DismissButton,
+  ButtonType,
+  ComponentColor,
+  FlexDirection,
+  JustifyContent,
+  DropdownMenuTheme,
+} from '@influxdata/clockface'
+
+// Types
+import {Threshold, ThresholdType} from 'src/types'
+import {LEVEL_COLORS} from 'src/alerting/constants'
+
+interface Props {
+  threshold: Threshold
+  removeLevel: () => void
+  changeThresholdType: (toType: ThresholdType, within?: boolean) => void
+}
+
+const OptionSelector = (threshold: Threshold) => {
+  if (threshold.type == 'greater') {
+    return 'is above'
+  }
+  if (threshold.type == 'lesser') {
+    return 'is below'
+  }
+  if (threshold.within) {
+    return 'is inside range'
+  }
+  return 'is outside range'
+}
+
+const ThresholdStatement: FC<Props> = ({
+  threshold,
+  children,
+  removeLevel,
+  changeThresholdType,
+}) => {
+  const dropdownOptions = [
+    'is above',
+    'is below',
+    'is inside range',
+    'is outside range',
+  ]
+
+  const levelColor: string = LEVEL_COLORS[threshold.level]
+  const selectedOption = OptionSelector(threshold)
+  const onChangeThresholdType = (option: string) => {
+    switch (option) {
+      case 'is above':
+        changeThresholdType('greater')
+        break
+      case 'is below':
+        changeThresholdType('lesser')
+        break
+      case 'is inside range':
+        changeThresholdType('range', true)
+        break
+      case 'is outside range':
+        changeThresholdType('range', false)
+        break
```

Comment:
Using the new `Dropdown` component would make this a bunch easier

Reply:
how come?

Reply:
you wouldn't need this switch statement for one :)